### DISCUSSION TITLE
Welcome to Pixelorama Discussions!

### DISCUSSION THREAD

**Discussion by OverloadedOrama** [Orama-Interactive/Pixelorama#396] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BentotSquared** [comment]
Hello I'm Bentot and I'm a young starting game developer that uses Godot to make games but can't really make games today because of full time schedule in school.  I use Pixelorama to create my game arts and graphics if its pixel art.

What do I do in this community?
I am just a member that wanted to see the development of Pixelorama and also to ask questions when I'm lost.
